src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ onTabSelected, tabs, selectedTabId }) => {
+  const setTab = tabs.find(el => el.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+
+          {tabs.map(tab => (
```

Comment:
Remove redundant spaces 
```suggestion
        <ul>
          {tabs.map(tab => (
```